charts/kube-prometheus-stack/values.yaml

```diff
@@ -60,6 +60,7 @@ defaultRules:
     nodeExporterRecording: true
     prometheus: true
     prometheusOperator: true
+    windows: true
```

Comment:
why should this be enabled by default?
windwos nodes are exoitc. 
this is not needed by everypne.

Reply:
the rules are only deployed if `defaultRules.rules.windows` (default `true`) AND `windowsMonitoring.enabled`  (default `false`) would be true